package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "iobroker.smartmeter",
-  "version": "3.2.0",
+  "version": "3.2.1-0",
```

Comment:
Please do not increase version  ... I use the release script!

Reply:
that's why I use the pre-releases tag you can now just make a release and it will be 3.2.1 ;)

Reply:
Yes but I see version increase a maintaners job and should (no longer) be in the PR ... ok for this time